smoke-test/src/main/java/org/opennms/smoketest/containers/OpenNMSContainer.java

```diff
@@ -56,12 +56,7 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import org.awaitility.core.ConditionTimeoutException;
-import org.opennms.smoketest.stacks.InternetProtocol;
-import org.opennms.smoketest.stacks.IpcStrategy;
-import org.opennms.smoketest.stacks.NetworkProtocol;
-import org.opennms.smoketest.stacks.OpenNMSProfile;
-import org.opennms.smoketest.stacks.StackModel;
-import org.opennms.smoketest.stacks.TimeSeriesStrategy;
+import org.opennms.smoketest.stacks.*;
```

Comment:
Can remove wildcard imports

Reply:
I cannot remove it as after all the imports that needs to be added, IDEA compress it to a wildcard instead of many imports.